features/admin.actions.v1/components/pre-update-password-action-config-form.tsx

```diff
@@ -169,6 +194,15 @@ const PreUpdatePasswordActionConfigForm: FunctionComponent<PreUpdatePasswordActi
     const handleSubmit = (
         values: PreUpdatePasswordActionConfigFormPropertyInterface,
         changedFields: PreUpdatePasswordActionConfigFormPropertyInterface) => {
+
+        let payloadRule: RuleWithoutIdInterface | RuleExecuteCollectionWithoutIdInterface | Record<string, never>;
```

Comment:
IMO, `rulePayload` would be better variable name here. WDYT?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7486/commits/34a926448e3cea96642355bfe473951501c89739